build/webpack/helpers.js

```diff
@@ -11,10 +9,8 @@ const makeCacheGroup = (name, priority, ...matches) => ({
     },
 });
 
-const publicPathFactory = (grunt) => () => (
-    (global.is_release || grunt.file.exists(PATHS.ROOT, 'scripts/CNAME') ? '' : '/binary-static') +
-    (global.branch ? `/${global.branch_prefix}${global.branch}` : '') +
-    '/js/'
+const publicPathFactory = () => () => (
+    `${global.branch ? `/binary-static/${global.branch_prefix}${global.branch}` : ''}/js/`
```

Comment:
Will this still work in `gh-pages` deploy?

Reply:
Yes tested on my gh-pages!

Reply:
yes, tested on my own gh-pages